File: src/main/golang/leetcode/P1365_How_Many_Numbers_Are_Smaller_Than_the_Current_Number.go
package leetcode

func smallerNumbersThanCurrent(nums []int) []int {
	result := make([]int, len(nums))

	for i, len := 0, len(nums); i < len; i++ {
		for j := 0; j < len; j++ {
			if i == j {
				continue
			}

			if nums[i] > nums[j] {
				result[i]++
			}
		}
	}

	return result
}

File: src/main/java/leetcode/P1365_How_Many_Numbers_Are_Smaller_Than_the_Current_Number.java
package leetcode;

public class P1365_How_Many_Numbers_Are_Smaller_Than_the_Current_Number {

    public int[] smallerNumbersThanCurrent(int[] nums) {
        int[] result = new int[nums.length];

        for (int i = 0, len = nums.length; i < len; i++) {
            for (int j = 0; j < len; j++) {
                if (i == j) continue;

                if (nums[i] > nums[j]) {
                    result[i]++;
                }
            }
        }

        return result;
    }

}
